2022-01-21 15:56:35,907 INFO    MainThread:2836431 [wandb_setup.py:_flush():71] setting env: {}
2022-01-21 15:56:35,908 INFO    MainThread:2836431 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-21 15:56:35,908 INFO    MainThread:2836431 [wandb_init.py:_log_setup():371] Logging user logs to /home/dogoulis/WORKSPACE/gan-prediction/experiments/wandb/run-20220121_155635-2ocielpu/logs/debug.log
2022-01-21 15:56:35,908 INFO    MainThread:2836431 [wandb_init.py:_log_setup():372] Logging internal logs to /home/dogoulis/WORKSPACE/gan-prediction/experiments/wandb/run-20220121_155635-2ocielpu/logs/debug-internal.log
2022-01-21 15:56:35,909 INFO    MainThread:2836431 [wandb_init.py:init():404] calling init triggers
2022-01-21 15:56:35,909 INFO    MainThread:2836431 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'Model-Type': 'resnet', 'GIQA': True, 'device': 'cuda', 'giqa': True, 'test_dir': '/home/dogoulis/WORKSPACE/gan-prediction/datasets/CSV/train_2k.csv', 'id': None, 'weights_dir': './epoch-0.pt'}
2022-01-21 15:56:35,909 INFO    MainThread:2836431 [wandb_init.py:init():449] starting backend
2022-01-21 15:56:35,909 INFO    MainThread:2836431 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-21 15:56:35,935 INFO    MainThread:2836431 [backend.py:ensure_launched():199] starting backend process...
2022-01-21 15:56:35,950 INFO    MainThread:2836431 [backend.py:ensure_launched():204] started backend process with pid: 2836472
2022-01-21 15:56:35,952 INFO    MainThread:2836431 [wandb_init.py:init():458] backend started and connected
2022-01-21 15:56:35,955 INFO    MainThread:2836431 [wandb_init.py:init():519] updated telemetry
2022-01-21 15:56:35,998 INFO    MainThread:2836431 [wandb_init.py:init():549] communicating current version
2022-01-21 15:56:37,743 INFO    MainThread:2836431 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-21 15:56:37,744 INFO    MainThread:2836431 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-21 15:56:38,049 INFO    MainThread:2836431 [wandb_init.py:init():592] starting run threads in backend
2022-01-21 15:56:43,057 INFO    MainThread:2836431 [wandb_run.py:_console_start():1816] atexit reg
2022-01-21 15:56:43,057 INFO    MainThread:2836431 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-01-21 15:56:43,058 INFO    MainThread:2836431 [wandb_run.py:_redirect():1695] Redirecting console.
2022-01-21 15:56:43,059 INFO    MainThread:2836431 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-21 15:56:43,060 INFO    MainThread:2836431 [wandb_init.py:init():619] run started, returning control to user process
2022-01-21 15:56:47,614 INFO    MainThread:2836431 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-21 15:56:47,614 INFO    MainThread:2836431 [wandb_run.py:_restore():1758] restore
2022-01-21 15:56:50,078 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 909
}

2022-01-21 15:56:50,207 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 909
}

2022-01-21 15:56:50,551 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 909
}

2022-01-21 15:56:51,097 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 21823
}

2022-01-21 15:56:51,200 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 22038
}

2022-01-21 15:56:51,302 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 22038
}

2022-01-21 15:56:51,405 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 22038
}

2022-01-21 15:56:51,507 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:51,610 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:51,713 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:51,817 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:51,919 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:52,216 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}

2022-01-21 15:56:52,864 INFO    MainThread:2836431 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22038
  total_bytes: 22038
}
local_info {
}

2022-01-21 15:56:54,068 INFO    MainThread:2836431 [wandb_run.py:_append_files():2184] logging synced files
